target/linux/ramips/dts/mt7621_asus_rp-ac56.dts

```diff
@@ -0,0 +1,244 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "asus,rp-ac56", "mediatek,mt7621-soc";
+	model = "Asus RP-AC56";
+
+	aliases {
+		led-boot = &led_power_orange;
+		led-failsafe = &led_power_orange;
+		led-running = &led_power_white;
+		led-upgrade = &led_power_blue;
+		label-mac-device = &gmac0;
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x4000000>;
+	};
```

Comment:
memory node is not required (and thus generally omitted), as memory is auto-detected.

Reply:
Done